check.py

```diff
@@ -191,26 +191,6 @@ def fix(x):
     if actual != expected:
       fail(actual, expected)
 
-print '\n[ checking binaryen-shell experimental testcases... ]\n'
```

Comment:
These are the tests that are now in llvm_autogenerated, which is added to the script along with the dot_s tests.